helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```
@@ -280,6 +280,7 @@ prometheusOperator:
 prometheus:
   prometheusSpec:
     replicas: {{ .Values.prometheus.replicas }}
+    topologySpreadConstraints: {{- toYaml .Values.prometheus.topologySpreadConstraints | nindent 8 }}
```

```suggestion
    topologySpreadConstraints: {{- toYaml .Values.prometheus.topologySpreadConstraints | nindent 6 }}
```
Should be indented 6 right?